src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/tools/ansyn.annotations.visualizer.ts

```diff
@@ -42,16 +42,16 @@ import { IOverlay } from '../../../../../overlays/models/overlay.model';
 import { selectTranslationData } from '../../../../../overlays/overlay-status/reducers/overlay-status.reducer';
 import { SetOverlayTranslationDataAction } from '../../../../../overlays/overlay-status/actions/overlay-status.actions';
 import { Actions, ofType } from '@ngrx/effects';
+import { GeometryObject } from 'geojson';
 
 // @dynamic
 @ImageryPlugin({
 	supported: [OpenLayersMap],
-	deps: [Store, Actions, OpenLayersProjectionService, OL_PLUGINS_CONFIG]
+	deps: [Store, Actions, OpenLayersProjectionService, OL_PLUGINS_CONFIG, VisualizersConfig]
 })
 export class AnsynAnnotationsVisualizer extends BaseImageryPlugin {
-	// TODO - rename (?) and bring value from config;
-	private isContinuousDrawingAllowedFromConfig = false;
-	private openLastDrawnAnnotationContextMenuFromConfig = false;
+	private isContinuousDrawingEnabled = false;
```

Comment:
isContinuousDrawingEnabled: bool as it is already being set in the constructor
change to protected